### ISSUE TITLE
False positive: Extensions that use 'Joom'

### ISSUE THREAD

**Issue by anibalsanchez** [joomla-extensions/jedchecker#163]
Hi,

As far as I know, Joom is not a registered trademark, and only the word "Joomla" is the actual trademark,

```sh
Extensions that use 'Joomla' or a derivative of Joomla! in the extension name ('Content - JoomHelper Media') need to be licensed by OSM
```

So, this extension name is not an error.

@dryabov Please, could you correct this false positive error.

**Comment by dryabov** [comment]
https://extensions.joomla.org/support/knowledgebase/submission-requirements/jed-entries-checklists/

> **List B1: Joomla Trademark Checklist**
> The following criteria is considered while approving a listing based on the use of the Joomla trademark and logo.
> ...
> Extensions that use the word "Joomla" or a **derivative** of the world "Joomla" in the extension name.

"Joom" seems to be a clear derivative of the word "Joomla"

**Comment by anibalsanchez** [comment]
The point is that it is not a registered trademark. So, OSM is not the owner of J, Joo, Joom.... someone could even use Jooml.

Just in case, I'm going to check with Claire from the TM Team.

**Comment by Llewellynvdm** [comment]
Okay so Claire has stated the following:

> they are allowed and have been for years, on the TM side,  the derivative text was discussed years ago and i thought it had been cobled. if they want to tell us they are doing it then its nice but not neccessary ....

Then further more she said:

> disclaimer - TM policy may not reflect the JED naming policy* you could add " if you are in doubt then please contact TM"  i would prefer to tell them to go away nicely than send a C&D notice

So for all practical purposes the derivative text are allowed for years now... and so we need to allow it here (in the jedchecker) also. On the other hand we have been working in internal review of outdated documentation... that is still ongoing and so we should get these updated soon enough.

**Comment by Llewellynvdm** [comment]
@dryabov can you make this adjustment? or shall I do it?

**Comment by dryabov** [comment]
@Llewellynvdm I'll prepare a patch tomorrow.

**Comment by dryabov** [comment]
@Llewellynvdm And what about domain names? Do we allow to use "joom" there as well?
